pkg/inst/worker/worker.R

```diff
@@ -1,5 +1,24 @@
 # Worker class
 
+# Utility function to get current system time in milliseconds
+# this code snippet borrowed from [R.utils](cran.r-project.org/package=R.utils)
+currentTimeMillis <- function() {
```

Comment:
Borrowing code from R.tuils may have license issue because it is LGPL-licensed. We'd better re-write the code?